Tools/autotest/common.py

```diff
@@ -2652,6 +2652,18 @@ def run_tests(self, tests):
 
         return True
 
+    def test_parameters(self):
```

Comment:
test_behaviour_of_misstotal_parameter?

Reply:
Added a comment and made a subtest instead.  Just wanted somewhere to throw misellaneous tests or our list of tests is going to get ridiculous.